### PR TITLE
[Outlook] (sideloading) Clarify sideloading instructions for Outlook for Mac

### PR DESCRIPTION
**Pull request by samantharamon**
Update the manual sideloading instructions for Outlook for Mac.

### CODE REVIEW

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 1]
```diff
@@ -93,13 +93,17 @@ The process for sideloading an add-in in Outlook on the web depends upon whether
     ![Outlook 2016 ribbon pointing to Get Add-ins button.](../images/outlook-sideload-desktop-store.png)
 
     > [!IMPORTANT]
-    > If you don't see the **Get Add-ins** button in your version of Outlook, select:
+    > If you don't see the **Get Add-ins** button in your version of Outlook, do one of the following:
     >
-    > - **Store** button on the ribbon, if available.
+    > - If you're using Outlook for Mac or configured your ribbon layout to Simplified Ribbon in Outlook for Windows, select the **See more items (`...`)** button from the ribbon, then select **Get Add-ins**.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:98]:
```suggestion
    > - If you're using Outlook on Mac or configured your ribbon layout to Simplified Ribbon in Outlook on Windows, select the **See more items (`...`)** button from the ribbon, then select **Get Add-ins**.
```

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 2]
```diff
@@ -93,13 +93,17 @@ The process for sideloading an add-in in Outlook on the web depends upon whether
...
     >
     >   OR
     >
-    > - **File** menu, then select the **Manage Add-ins** button on the **Info** tab to open the **Add-ins** dialog in Outlook on the web.<br>You can see more about the web experience in the previous section [Sideload an add-in in Outlook on the web](#outlook-on-the-web).
+    > - Select the **Store** button on the ribbon, if available.
+    >
+    >   OR
+    >
+    > - For Outlook for Windows, select the **File** menu, then select the **Manage Add-ins** button on the **Info** tab to open the **Add-ins** dialog in Outlook on the web. You can see more about the web experience in the previous section [Sideload an add-in in Outlook on the web](#outlook-on-the-web).
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:106]:
```suggestion
    > - For Outlook on Windows, select the **File** menu, then select the **Manage Add-ins** button on the **Info** tab to open the **Add-ins** dialog in Outlook on the web. You can see more about the web experience in the previous section [Sideload an add-in in Outlook on the web](#outlook-on-the-web).
```

File: docs/outlook/sideload-outlook-add-ins-for-testing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samantharamon) [reply docs/outlook/sideload-outlook-add-ins-for-testing.md]: It's the text that appears when hovering over the ellipsis button in the Outlook on Mac client.

[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/outlook/sideload-outlook-add-ins-for-testing.md]: That's not the text I'm seeing in Windows so you might need to only mention the ellipsis. Then, if you want to, you could then mention the different text for Mac/Windows. The only thing is I'm not sure if that's the hover text that was included from the beginning.

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
I like the tabs idea. Included suggestion for taking it a bit further. Also curious about why you opted to use 2 versions of the tabs (non-nested and nested, as it were)...?

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 3]
```diff
@@ -84,22 +84,38 @@ The process for sideloading an add-in in Outlook on the web depends upon whether
...
-    > If you don't see the **Get Add-ins** button in your version of Outlook, select:
+    > If you don't see the **Get Add-ins** button in your version of Outlook, do one of the following:
+    >
+    > - If you're using Outlook on Mac or configured your ribbon layout to Simplified Ribbon in Outlook on Windows, select the ellipsis button (`...`) from the ribbon, then select **Get Add-ins**.
+    >
+    >   # [Windows](#tab/nestedWindows)
+    >
+    >   ![Outlook on Windows pointing to the Get Add-ins button from the ellipsis button.](../images/outlook-sideload-simplified-ribbon.png)
+    >
+    >   # [Mac](#tab/nestedMac)
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply docs/outlook/sideload-outlook-add-ins-for-testing.md:112]: That's a great idea, and it reads better, too! Thanks for that.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**